### DISCUSSION TITLE
Geting wrong styling for Heading component

### DISCUSSION THREAD

**Discussion by roman-kulpa** [twilio-labs/paste#828] [Q&A] [answered]
For the following component:
```
 <Heading as="h1" variant="heading10">test</Heading>
```
I'm getting the following generated css:
```
.css-lqdz95 {
    margin: 0;
    padding: 0;
    margin-bottom: space70;
    font-size: fontSize90;
    font-weight: fontWeightSemibold;
    line-height: lineHeight90;
    display: block;
    color: colorText;
}
```
what could be the reason for the wrong css values?

**Comment by TheSisb** [comment, marked as answer]
Hi @roman-kulpa,

Have you made sure to wrap your app in our ThemeProvider package?  Instructions can be found here: https://paste.twilio.design/getting-started/engineering#usage

**Reply by roman-kulpa** [reply]
Thanks `ThemeProvider` resolved the issue
